t/plugin/consumer-restriction.t

```diff
@@ -1436,3 +1436,254 @@ passed
 GET /t
 --- response_body
 passed
+
+
+
+=== TEST 54: create consumer group(group1)
```

Comment:
Could you move the new tests to a new file?

> When the test file is too large, for example > 800 lines, you should split it to a new file. Please take a look at t/plugin/limit-conn.t and t/plugin/limit-conn2.t.

https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#check-code-style-and-test-case-style

Reply:
done.